src/test/java/com/khoros/twitterapp/StatusUpdateResourceTest.java

```diff
@@ -39,12 +48,33 @@ public void setup() {
         } catch (TwitterException e) {
             Assert.fail("Test failed due to Twitter Exception: " + e.getMessage());
         }
-    }
 
-    @Test
-    public void statusUpdateTest() {
-        // expectedResponse = Response.status(200).entity(statusEntity).build();
         Assert.assertEquals(200, statusResource.postStatus(exampleText).getStatus());
         Assert.assertEquals(statusEntity, statusResource.postStatus(exampleText).getEntity());
     }
+
+    @Test
+    public void statusLengthZeroTest() {
+        exampleText = "";
+
+        Assert.assertEquals(403, statusResource.postStatus(exampleText).getStatus());
+        Assert.assertEquals("No tweet text entered.", statusResource.postStatus(exampleText).getEntity());
+    }
+
+    @Test
+    public void statusLengthLongTest() {
+        exampleText = "hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello hello";
+
+        Assert.assertEquals(403, statusResource.postStatus(exampleText).getStatus());
+        Assert.assertEquals("Tweet text surpassed 280 characters.", statusResource.postStatus(exampleText).getEntity());
```

Comment:
reference the text from a shared variable vs copying/pasting from the resource class